JavaAssignments/src/amarja/Arraycontnum.java

```diff
@@ -0,0 +1,34 @@
+package amarja;
+import java.util.*;
+//Question 3 Program to find array contains number 
+public class Arraycontnum {
+	boolean findnum(int arr[],int num)
+	{	boolean flag=false;
+		for (int index=0;index<arr.length;index++)
+		{
+			if (arr[index]==num)
```

Comment:
Once you find the number, you should break this loop.
Consider a scenario where there are 100 elements and you find first element as your number then also loop will iterate 99 time extra which is not required.